images/plbase/plbase-install.sh

```diff
@@ -45,45 +45,17 @@ echo "setting up postgres..."
 mkdir /var/postgres && chown postgres:postgres /var/postgres
 su postgres -c "initdb -D /var/postgres"
 
+# TODO: use standard OS Python installation? The only reason we switched to Conda
+# was to support R and `rpy2`, but now that we've removed those, we might not
+# get any benefit from Conda.
```

Comment:
AL2023 does support Python 3.11 as a separate package to install: https://docs.aws.amazon.com/linux/al2023/ug/python.html. Of course, I'm happy to stick with Conda for now; I certainly don't want to change that in this PR (since I want to minimize the number of changes here).